<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1569</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magnetometry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>741</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutTimeSeriesCanvas"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>911</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2DMappingCanvas"/>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>501</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">File Tree</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="layerWidget">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>10</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Layers:</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutTimeSeriesCanvas_2"/>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>160</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoothing window size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>220</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Ambient estimation window size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_smoothingWindow">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>190</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ambientWindow">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>240</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>270</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Gridding sampling rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nthSelectWindow">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>300</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1569</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Survey">
     <property name="title">
      <string>Import Survey</string>
     </property>
     <addaction name="actionFrom_BOB_CSV"/>
     <addaction name="actionFrom_Sealink_Folder"/>
     <addaction name="separator"/>
     <addaction name="actionFrom_Custom_CSV"/>
    </widget>
    <widget class="QMenu" name="menuExport_Grid">
     <property name="title">
      <string>Export Grid</string>
     </property>
     <addaction name="actionCSV"/>
     <addaction name="actionGeoTiff"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="menuOpen_Survey"/>
    <addaction name="separator"/>
    <addaction name="menuExport_Grid"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExperimental">
    <property name="title">
     <string>Experimental</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExperimental"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionDinuralCorrection"/>
   <addaction name="actionSmoothSurvey"/>
   <addaction name="separator"/>
   <addaction name="actioncalcResiduals"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionDrawSelect"/>
   <addaction name="actionRemoveOutlier"/>
   <addaction name="actionDraw1D"/>
  </widget>
  <action name="actionblubb">
   <property name="text">
    <string>blubb</string>
   </property>
  </action>
  <action name="actionDinuralCorrection">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/magnetic-icon.png</normaloff>:/icons/magnetic-icon.png</iconset>
   </property>
   <property name="text">
    <string>DinuralCorrection</string>
   </property>
   <property name="toolTip">
    <string>Diurnal correction</string>
   </property>
  </action>
  <action name="actionSmoothSurvey">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/improvement-performance-icon.png</normaloff>:/icons/improvement-performance-icon.png</iconset>
   </property>
   <property name="text">
    <string>SmoothSurvey</string>
   </property>
  </action>
  <action name="actioncalcResiduals">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/calculator-icon.png</normaloff>:/icons/calculator-icon.png</iconset>
   </property>
   <property name="text">
    <string>calcResiduals</string>
   </property>
   <property name="toolTip">
    <string>Calculate residuals based on ambient field</string>
   </property>
  </action>
  <action name="actionSomething_Else">
   <property name="text">
    <string>Something Else</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionFrom_BOB_CSV">
   <property name="text">
    <string>From BOB CSV</string>
   </property>
  </action>
  <action name="actionFrom_Sealink_Folder">
   <property name="text">
    <string>From Sealink Folder</string>
   </property>
  </action>
  <action name="actionFrom_Custom_CSV">
   <property name="text">
    <string>From Custom CSV</string>
   </property>
  </action>
  <action name="actionDrawSelect">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/drawingIcon.png</normaloff>:/icons/drawingIcon.png</iconset>
   </property>
   <property name="text">
    <string>DrawSelect</string>
   </property>
  </action>
  <action name="actionRemoveOutlier">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/eraser-icon.png</normaloff>:/icons/eraser-icon.png</iconset>
   </property>
   <property name="text">
    <string>RemoveOutlier</string>
   </property>
  </action>
  <action name="actionDraw1D">
   <property name="icon">
    <iconset resource="icons/resources.qrc">
     <normaloff>:/icons/area-chart-icon.png</normaloff>:/icons/area-chart-icon.png</iconset>
   </property>
   <property name="text">
    <string>Draw1D</string>
   </property>
   <property name="toolTip">
    <string>Draw a timeseries representation of selected surveys</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionGeoTiff">
   <property name="text">
    <string>GeoTiff</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
